File: Js/Contagem.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Digite o valor de N: ', (N) => {
    rl.question('Digite o primeiro número: ', (primeiroNumero) => {
        rl.question('Digite o conjunto de dados separado por espaços: ', (dadosInput) => {
            const dados = dadosInput.split(' ').map(Number);
            let count = 0;

            for (let i = 0; i < N; i++) {
                const numero = dados[i];
                if (numero >= primeiroNumero && numero <= N) {
                    count++;
                }
            }

            console.log(`Quantidade de números no intervalo: ${count}`);
            rl.close();
        });
    });
});

File: Java/Contagem.java
package Java;
import java.util.Scanner;

public class Contagem {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o valor de N: ");
        int N = scanner.nextInt();

        System.out.print("Digite o primeiro número: ");
        int primeiroNumero = scanner.nextInt();

        System.out.print("Digite o conjunto de dados separado por espaços: ");
        int count = 0;

        for (int i = 0; i < N; i++) {
            int numero = scanner.nextInt();
            if (numero >= primeiroNumero && numero <= N) {
                count++;
            }
        }

        System.out.println("Quantidade de números no intervalo: " + count);
        scanner.close();
    }
}